models/P2PClient/index.js

```diff
@@ -0,0 +1,18 @@
+/*
+State:
+star = peer-star instance
+presenter = simple-peer to Presenter
+connection = true if connected to presenter, false if not
+username = name of the user (chosen by the user)
+ */
+
+module.exports = globalConfig => ({
+  namespace: 'p2c',
+  effects: require('./effects')(globalConfig),
```

Comment:
Okay as far as I took a look into the existing choo modules the effects and subscriptions were mostly directly defined within the export object. We can change this of course. Again for orientation i took a look at another repo, where things were handled this way. For consistency I agree we should handle it with a const.